lcls-twincat-common-components/Library/Devices/PPM/FB_PPM_PowerMeter.TcPOU

```diff
@@ -288,31 +278,6 @@ END_IF
 fPulseWattage := SEL(fResponsivity <> 0, 0, (fVoltage - fBackgroundVoltage) / fResponsivity);
 fCalibMJ := SEL(fFrequency <> 0, -9999, fPulseWattage / fFrequency);
 
-// FF in case voltage is high enough to damage power meter based on wavelengths
-{IF defined (K)}
-    CASE REAL_TO_UINT(PMPS_GVL.stCurrentBeamParameters.neV) OF
-        0 .. uSoftWavelengthEdge1L: bOverAllowableEnergy := fCalibMJ > 2;
-        uSoftWavelengthEdge1 .. uSoftWavelengthEdge2L : bOverAllowableEnergy := fCalibMJ > 2;
-        uSoftWavelengthEdge2 .. uSoftWavelengthEdge3L: bOverAllowableEnergy := fCalibMJ > 4;
-        uSoftWavelengthEdge3 .. uSoftWavelengthEdge4L: bOverAllowableEnergy := fCalibMJ > 0.5;
-        uSoftWavelengthEdge4 .. 65535: bOverAllowableEnergy := fCalibMJ > 1;
-    END_CASE
-{ELSIF defined (L)}
-    CASE REAL_TO_UINT(PMPS_GVL.stCurrentBeamParameters.neV) OF
-        uHardWavelengthEdge1 .. uHardWavelengthEdge2L: bOverAllowableEnergy := fCalibMJ > 10;
-        uHardWavelengthEdge2 .. uHardWavelengthEdge3L: bOverAllowableEnergy := fCalibMJ > 4;
-        uHardWavelengthEdge3 .. 65535: bOverAllowableEnergy := fCalibMJ > 10;
-    END_CASE
-{END_IF}
-IF eEnumGet = E_PPM_States.POWERMETER AND bOverAllowableEnergy THEN
-    uOverCounter := MIN(uOverCounter + 1, 10);
-ELSE
-    uOverCounter := 0;
-END_IF
-FF(i_xOK := eEnumGet <> E_PPM_States.POWERMETER OR uOverCounter < 10,
-    i_DevName := sDeviceName,
-    io_fbFFHWO := fbFFHWO);
```

Comment:
I'm sad to see this bit of code go. I've learned some valuable lessons about backcompat wrt PMPS from this at least.